Documentation/devicetree/bindings/net/nfc/nxp,nci.yaml

```
@@ -24,6 +24,9 @@ properties:
   firmware-gpios:
     description: Output GPIO pin used to enter firmware download mode
 
+  vddio-supply:
+    description: Optional regulator used for some PN547 controller
```

```suggestion
  pvdd-supply:
    description: Optional regulator used for some PN544 and PN547 controller
```
- Typo. should be `pvdd-supply`
- Actually it's "pad supply"